ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_layers.py

```diff
@@ -1276,3 +1322,25 @@ def test_rfftn(
         axes=axes,
         norm=norm,
     )
+
+
+@handle_test(
+    fn_tree="functional.ivy.experimental.sliding_window",
+    all_args=_sliding_window_helper(),
+    test_with_out=st.just(False),
+    ground_truth_backend="jax",
+)
```

Comment:
In the CI checks I see some failures that I think are related to the missing array and container methods. You need to either add the needed code, or exclude these cases from `handle_test` and add a ToDo.

Other than that, looks great overall!